lsp.code-workspace

```diff
@@ -18,9 +18,11 @@
             "**/out": true,
         },
         "rubyLsp.bypassTypechecker": true,
-        "rubyLsp.excludedPatterns": [
```

Comment:
I thought this has been updated in #2319?

Reply:
Add, bad rebase, will fix.